README.md

```diff
@@ -238,13 +270,23 @@ def db_url():
     return "postgresql://postgres@localhost/test_database"
 ```
 
+### `async_sqlalchemy_url`
+
+DB url to use when using `asyncio` support. Defaults to `db_url` fixture with
+`postgresql+asyncpg://` scheme.
 
-### `session`
 
-Sqla session to create db fixture:
+### `session` & `async_session`
+
+Sqla sessions to create db fixture:
 * All changes done at test setup or during the test are rollbacked at test tear down;
 * No record will actually be written in the database;
-* Changes in one session need to be committed to be available from other sessions;
+* Changes in one regular session need to be committed to be available from other regular
+  sessions;
+* Changes in one async session need to be committed to be available from other async
+  sessions;
+* Changes from regular sessions are not available from `async` session and vice-versa
```

Comment:
```suggestion
* **Changes from regular sessions are not available from `async` session and vice-versa**
```

Reply:
WDYM? That uncommitted changes are not shared amongst async/sync sessions?

Reply:
when using fixture, even committed change can't be seen as nothing is actually committed/written to db.

So if your code under test uses an async session, it has to be tested using the `async_session` fixture and vice versa

I guess this should be rephrased/documented.

Reply:
Ah my bad! With the diff view, I didn't see that this comment was under the 'testing' section, it makes sense now :)